src/set_theory/game/pgame.lean

```diff
@@ -261,287 +249,284 @@ instance : has_one pgame := ⟨⟨punit, pempty, λ _, 0, pempty.elim⟩⟩
-by { cases x, rw mk_lt_mk, exact λ h, or.inl ⟨_, h⟩ }
+theorem lf_of_lt {x y : pgame} (h : x < y) : x ⧏ y :=
+(lt_iff_le_and_lf.1 h).2
 
-theorem move_left_lt_of_le {x y : pgame} {i} :
-  x ≤ y → x.move_left i < y :=
-by { cases x, exact lt_of_le_mk }
+/-- The definition of `x ≤ y` on pre-games, in terms of `≤` two moves later. -/
+theorem le_def {x y : pgame} : x ≤ y ↔
+  (∀ i, (∃ i', x.move_left i ≤ y.move_left i')  ∨ ∃ j, (x.move_left i).move_right j ≤ y) ∧
+   ∀ j, (∃ i, x ≤ (y.move_right j).move_left i) ∨ ∃ j', x.move_right j' ≤ y.move_right j :=
+by { rw le_def_lf, conv { to_lhs, simp only [lf_def_le] } }
 
-theorem lt_move_right_of_le {x y : pgame} {i} :
-  x ≤ y → x < y.move_right i :=
-by { cases y, exact lt_of_mk_le }
+/-- The definition of `x ⧏ y` on pre-games, in terms of `⧏` two moves later. -/
+theorem lf_def {x y : pgame} : x ⧏ y ↔
+  (∃ i, (∀ i', x.move_left i' ⧏ y.move_left i)  ∧ ∀ j, x ⧏ (y.move_left i).move_right j) ∨
+   ∃ j, (∀ i, (x.move_right j).move_left i ⧏ y) ∧ ∀ j', x.move_right j ⧏ y.move_right j' :=
+by { rw lf_def_le, conv { to_lhs, simp only [le_def_lf] } }
 
-theorem lt_of_move_right_le {x y : pgame} {i} :
-  x.move_right i ≤ y → x < y :=
-by { cases x, rw move_right_mk, exact mk_lt_of_le }
+/-- The definition of `0 ≤ x` on pre-games, in terms of `0 ⧏`. -/
+theorem zero_le_lf {x : pgame} : 0 ≤ x ↔ ∀ j, 0 ⧏ x.move_right j :=
+by { rw le_def_lf, dsimp, simp }
 
-theorem lt_of_le_move_left {x y : pgame} {i} :
-  x ≤ y.move_left i → x < y :=
-by { cases y, rw move_left_mk, exact lt_mk_of_le }
+/-- The definition of `x ≤ 0` on pre-games, in terms of `⧏ 0`. -/
+theorem le_zero_lf {x : pgame} : x ≤ 0 ↔ ∀ i, x.move_left i ⧏ 0 :=
+by { rw le_def_lf, dsimp, simp }
 
-private theorem not_le_lt {x y : pgame} :
-  (¬ x ≤ y ↔ y < x) ∧ (¬ x < y ↔ y ≤ x) :=
-begin
-  induction x with xl xr xL xR IHxl IHxr generalizing y,
-  induction y with yl yr yL yR IHyl IHyr,
-  classical,
-  simp only [mk_le_mk, mk_lt_mk,
-    not_and_distrib, not_or_distrib, not_forall, not_exists,
-    and_comm, or_comm, IHxl, IHxr, IHyl, IHyr, iff_self, and_self]
-end
+/-- The definition of `0 ⧏ x` on pre-games, in terms of `0 ≤`. -/
+theorem zero_lf_le {x : pgame} : 0 ⧏ x ↔ ∃ i, 0 ≤ x.move_left i :=
+by { rw lf_def_le, dsimp, simp }
 
-@[simp] theorem not_le {x y : pgame} : ¬ x ≤ y ↔ y < x := not_le_lt.1
-@[simp] theorem not_lt {x y : pgame} : ¬ x < y ↔ y ≤ x := not_le_lt.2
+/-- The definition of `x ⧏ 0` on pre-games, in terms of `≤ 0`. -/
+theorem lf_zero_le {x : pgame} : x ⧏ 0 ↔ ∃ j, x.move_right j ≤ 0 :=
+by { rw lf_def_le, dsimp, simp }
 
-@[refl] protected theorem le_rfl : ∀ {x : pgame}, x ≤ x
-| ⟨l, r, L, R⟩ := by rw mk_le_mk; exact
-⟨λ i, lt_mk_of_le le_rfl, λ i, mk_lt_of_le le_rfl⟩
+/-- The definition of `0 ≤ x` on pre-games, in terms of `0 ≤` two moves later. -/
+theorem zero_le {x : pgame} : 0 ≤ x ↔ ∀ j, ∃ i, 0 ≤ (x.move_right j).move_left i :=
+by { rw le_def, dsimp, simp }
 
-protected theorem le_refl (x : pgame) : x ≤ x :=
-pgame.le_rfl
```

Comment:
These are removed but I think it's OK because we now have the `preorder` instance.

Reply:
I don't think it's only ok, but strictly better. Otherwise, we end up with duplicate theorems.